Docs: Add information about 'WITH' in CREATE GROUP (#13717)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/yql-core/syntax/create-group.md

# CREATE GROUP

> Creates a group with the specified name with the option to specify the users who belong to this group.

Reviewer: ```suggestion
Creates a [group](../../../../concepts/glossary.md#access-group) with the specified name. Optionally, you can specify a list of [users](../../../../concepts/glossary.md#access-user) to add to the group.
```

## Syntax

```yql
CREATE GROUP group_name [ WITH user_name [ , user_name [ ... ]] [ , ] ]
```

### Parameters

* `group_name`: The name of the group. It may contain lowercase Latin letters and digits.

> * `user_name`: The name of the user who will become a member of the group after its creation (note that you can specify multiple users). It may contain lowercase Latin letters and digits.

Reviewer: ```suggestion
* `user_name`: The name of the user to add to the group after its creation. The name may contain lowercase Latin letters and digits.

    {% note info %}
    You can specify multiple users in a comma-separated list.
    {% endnote %}

```

## Examples

```yql
CREATE GROUP group1;
```

```yql
CREATE GROUP group2 WITH USER user1;
```

```yql

> CREATE GROUP group3 WITH USER user1, user2,;

Reviewer: ```suggestion
CREATE GROUP group3 WITH USER user1, user2;
```
